monitors/index.js

```diff
@@ -145,6 +145,9 @@ async function run(
       await medianizerPriceFeed.update();
       // 4.b Check for synthetic peg deviation
       await syntheticPegMonitor.checkPriceDeviation();
+      // 4.c Check for price feed volatility
+      await syntheticPegMonitor.checkPegVolatility();
```

Comment:
@chrismaree I added both monitors to the `index.js` for monitors here, wdyt about this?

Reply:
looks awesome. I like how you've set this up. it fits well within our larger monitor infra.